pusher/src/Services/LocalCompanionSevice.ts

```diff
@@ -1,14 +1,20 @@
-export interface CompanionResourceDescriptionInterface {
-    name: string;
-    img: string;
-    behaviour: "dog" | "cat";
-}
+import { CompanionServiceList } from "./CompanionServiceList";
+import { CompanionList } from "../Messages/JsonMessages/CompanionTextures";
 
-export const COMPANION_RESOURCES: CompanionResourceDescriptionInterface[] = [
+const defaultCompanionList: CompanionList = [
```

Comment:
I think the companion list must be a json file into the pusher data folder like wokas.